File: python/FizzBuzz.py
#実行方法
#プログラムファイルがあるディレクトリにて python3 FizzBuzz.pyをコンソールにて実行


    #ループ処理rangeが範囲
for i in range(1,101):
    #条件式
    if(i % 3 == 0 | i % 5 == 0):
        result = "FizzBuzz" 
    elif(i % 3 == 0):
        result = "Fizz"
    elif(i % 5 == 0):
        result = "Buzz"
    else:
        result = i
    #出力
    print(result)    

File: Rust/FizzBuzz.rs
/*
実行方法
コンソールにてプログラムがあるディレクトリに移動する
コンソールにて"rustc FizzBuzz.rs"を実行する
FizzBuzzファイルが作成される
コンソールにて"./FizzBuzz"を実行する       
*/


fn main() {
    for i in 1..100 {
        if i % 3 == 0 && i % 5 == 0{
            println!("FizzBuzz");
        }else if i % 3 == 0{
            println!("Fizz");
        }else if i % 5 == 0{
            println!("Buzz");
        }else{
            println!("{}",i);
        }
    }
}